addons/project/models/project.py

```diff
@@ -1020,7 +1018,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     legend_blocked = fields.Char(related='stage_id.legend_blocked', string='Kanban Blocked Explanation', readonly=True, related_sudo=False)
     legend_done = fields.Char(related='stage_id.legend_done', string='Kanban Valid Explanation', readonly=True, related_sudo=False)
     legend_normal = fields.Char(related='stage_id.legend_normal', string='Kanban Ongoing Explanation', readonly=True, related_sudo=False)
-    is_closed = fields.Boolean(related="stage_id.fold", string="Closing Stage", related_sudo=False, help="Folded in Kanban stages are closing stages.")
+    is_closed = fields.Boolean(related="stage_id.fold", string="Closing Stage", store=True, index=True, related_sudo=False, help="Folded in Kanban stages are closing stages.")
```

Comment:
Does it really need to be stored? Doesn't use `auto_join=True` do the trick?

Reply:
Hello @KangOl 👋🏼 
Are you still against it after having seen the explain plan I provided you with or can we go forward with this PR?